prep-css-box-model/style.css

```diff
@@ -0,0 +1,10 @@
+body {
+  text-align: center;
+}
+
+button {
+  background-color: red;
+  color: white;
+
+  /* 👇 YOUR CODE HERE 👇 */
```

Comment:
Hi Isaac! We want to put the CSS code in this `style.css` file.

Reply:
![prep-css-box-model2](https://user-images.githubusercontent.com/85271794/121569943-32f47d00-c9d6-11eb-9fef-ad058b9f2e19.png)